crates/db_schema/src/schema.rs

```diff
@@ -1,18 +1,73 @@
-table! {
+// @generated automatically by Diesel CLI.
+
+pub mod sql_types {
+  #[derive(diesel::sql_types::SqlType)]
+  #[diesel(postgres_type(name = "listing_type_enum"))]
+  pub struct ListingTypeEnum;
+
+  #[derive(diesel::sql_types::SqlType)]
+  #[diesel(postgres_type(name = "registration_mode_enum"))]
+  pub struct RegistrationModeEnum;
+
+  #[derive(diesel::sql_types::SqlType)]
+  #[diesel(postgres_type(name = "sort_type_enum"))]
+  pub struct SortTypeEnum;
+}
+
+diesel::table! {
     activity (id) {
         id -> Int4,
         data -> Jsonb,
         local -> Bool,
         published -> Timestamp,
         updated -> Nullable<Timestamp>,
         ap_id -> Text,
-        sensitive -> Nullable<Bool>,
+        sensitive -> Bool,
+    }
+}
+
+diesel::table! {
+    admin_purge_comment (id) {
+        id -> Int4,
+        admin_person_id -> Int4,
+        post_id -> Int4,
+        reason -> Nullable<Text>,
+        when_ -> Timestamp,
+    }
+}
+
+diesel::table! {
+    admin_purge_community (id) {
+        id -> Int4,
+        admin_person_id -> Int4,
+        reason -> Nullable<Text>,
+        when_ -> Timestamp,
+    }
+}
+
+diesel::table! {
+    admin_purge_person (id) {
+        id -> Int4,
+        admin_person_id -> Int4,
+        reason -> Nullable<Text>,
+        when_ -> Timestamp,
     }
 }
 
-table! {
-  use diesel_ltree::sql_types::Ltree;
-  use diesel::sql_types::*;
+diesel::table! {
+    admin_purge_post (id) {
+        id -> Int4,
+        admin_person_id -> Int4,
+        community_id -> Int4,
+        reason -> Nullable<Text>,
+        when_ -> Timestamp,
+    }
+}
+
+diesel::table! {
+    use diesel::sql_types::*;
+    // TODO this must be manually replaced unfortunately
+    use diesel_ltree::sql_types::Ltree;
```

Comment:
It looks like `import_types` field in diesel.toml could take care of this.

https://diesel.rs/guides/configuring-diesel-cli.html

Reply:
Tried it and couldn't get it to work, but found a way with a patch file.